cmd/sealer/cmd/root.go

```diff
@@ -98,19 +100,16 @@ func initConfig() {
 
 	viper.AutomaticEnv() // read in environment variables that match
 
-	if rootOpt.debugModeOn {
-		logrus.SetLevel(logrus.DebugLevel)
-	}
-
-	logrus.SetFormatter(&logrus.TextFormatter{
-		DisableColors: rootOpt.colorMode == colorModeNever,
+	err := logger.Init(logger.LogOptions{
+		WriteToDisk:          rootOpt.writeToDisk,
+		Verbose:              rootOpt.debugModeOn,
+		RemoteLoggerURL:      rootOpt.remoteLoggerURL,
+		RemoteLoggerTaskName: rootOpt.remoteLoggerTaskName,
+		DisableColor:         rootOpt.colorMode == colorModeNever,
 	})
 
-	if rootOpt.remoteLoggerURL != "" {
-		rl, err := remotelogger.NewRemoteLogHook(rootOpt.remoteLoggerURL, rootOpt.remoteLoggerTaskName)
-		if err != nil {
-			panic(err)
-		}
-		logrus.AddHook(rl)
+	//it is a logger init error which does not affect the sealer main process. thus, marked it as tolerable error.
+	if err != nil {
+		fmt.Println(err)
```

Comment:
we should not log error just in stdout using `fmt.Println`, at least in stderr. @kakaZhou719

Reply:
> we should not log error just in stdout using `fmt.Println`, at least in stderr. @kakaZhou719

using `os.Stderr.WriteString` to write init error .

Reply:
fmt.Fprintf(os.Stderr, "failed to init logger: %v\n", err)